### PR TITLE
fixes terrible alert by making it announce area instead of turf (and a bad span thing)

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

"Network Alert: Hacking attempt detected in the floor." also the period is left out of the span

## Why It's Good For The Game

Naming the area will at least give the AI something to go off of

### CODE REVIEW

**Comment by cacogen** [comment]
this has been like this forever

File: code/modules/ninja/suit/ninjaDrainAct.dm [hunk 1]
```diff
@@ -115,8 +115,8 @@ They *could* go in their appropriate files, but this is supposed to be modular
 			update_icon()
 
 /obj/machinery/proc/AI_notify_hack()
-	var/turf/location = get_turf(src)
-	var/alertstr = "<span class='userdanger'>Network Alert: Hacking attempt detected[location?" in [location]":". Unable to pinpoint location"]</span>."
+	var/area/location = get_area(src)
+	var/alertstr = "<span class='userdanger'>Network Alert: Hacking attempt detected[location?" in [location]":". Unable to pinpoint location"].</span>"
```
Comment (ShizCalev) [inline code/modules/ninja/suit/ninjaDrainAct.dm]: use `get_area_name(src, TRUE)` instead

Comment (ShizCalev) [inline code/modules/ninja/suit/ninjaDrainAct.dm]: var/alertstr = "<span class='userdanger'>Network Alert: Hacking attempt detected[get_area()?" in [get_area_name(src, TRUE)]":". Unable to pinpoint location"].</span>"

Comment (ShizCalev) [reply code/modules/ninja/suit/ninjaDrainAct.dm]: and remove the cached var

**Review by spookydonut — APPROVED** [review]

**Pull request merged**